contrib/examples/example_add_bot/setup.py

```diff
@@ -0,0 +1,39 @@
+from setuptools import find_packages, setup
+
+setup(
+    name='intelmqExampleAddBot',
+    version=3.1,  # noqa: F821
+    maintainer='Sebastian Wagner',
+    maintainer_email='intelmq-dev@lists.cert.at',
+    python_requires='>=3.7',
```

Comment:
```suggestion
```
intelmq itself requires python >= 3.7 currently, that may not be true for bots. as the bots require the core library anyway, the requirement is active too without listing it here